lib/sprockets/media_query_combiner/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sprockets
   module MediaQueryCombiner
-    VERSION = "0.0.8"
```

Comment:
Can you leave this out? I'll bump the version myself (typically you want to leave these out of pulls as the maintainer may want to bundle multiple things in one version... don't worry though, I'll release this as 0.0.9 as soon as its merged)